rhel8/profiles/ism_o.profile

```diff
@@ -0,0 +1,145 @@
+documentation_complete: true
+
+title: 'Australian Cyber Security Centre (ACSC) Information Security Manual (ISM) Official'
+
+description: |-
+  This profile contains configuration checks for Red Hat Enterprise Linux 8
+  that align to the Australian Cyber Security Centre (ACSC) Information Security Manual (ISM)
+  with the Attorney-General’s Department (AGD)’s applicability marking of OFFICIAL.
+
+  A overview and list of Cyber security guidelines of the 
+  Information Security Manual can be found at the ACSC website:
+
+  https://www.cyber.gov.au/ism
+
+extends: e8
+
+selections:
+
+  ## Operating system configuration
+  ## Identifiers 1491
+  - no_shelllogin_for_systemaccounts
+
+  ## Local administrator accounts
+  ## Identifiers 1410
+  - accounts_password_all_shadowed
+
+  ## Content filtering & Anti virus
+  ## Identifiers 1341 / 1034 / 1417 / 1288
+  - package_aide_installed
+
+  ## Software firewall
+  ## Identifiers 1416
+  - configure_firewalld_ports
+  - configure_firewalld_rate_limiting
+  - firewalld_sshd_port_enabled
+  - set_firewalld_default_zone
+
+  ## Endpoint device control software
+  ## Identifiers 1418
+  - package_usbguard_installed
+  - service_usbguard_enabled
+  - usbguard_rules_not_empty_not_missing
```

Comment:
This is also not a rule, it is just a check used in different usbguard rules. Please see rules in the services/usbguard directory.

Reply:
Removed @vojtapolasek https://github.com/wcushen/content/commit/4ff7f4596bf0c130c05c2e4601cc8067ed7965e3